_posts/2024-09-05-releasing-apps.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Releasing apps at M&S
+published: true
+author: 
+  - name: Jamie Sanson
+    github: jamiesanson
+date: 2024-09-05
+categories:
+  - mobile
+tags:
+  - mobile
+  - platform
+banner:
+  image: digital-design-language.png
+---
+
+At Marks & Spencer we ship many apps all the time. Our customer facing apps are growing by the day, and our old, time-consuming fortnightly release process was starting to show its age.
```

Comment:
You could mention the number of apps. Like I say below, I think the portability factor of the new workflow is super impressive and readers will want to read that. If you tease it at the top they will read further